### DISCUSSION TITLE
Is the "Changing the colors" page outdated?

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#3378] [Q&A]
The "Changing the colors" page does mention that you can override colours using something alongside this:
```css
:root {
    --md-primary-fg-color: #123456;
}
```

The issue with that is, that this override does sometimes not work.

In my case did the above not work, no matter what I did. Only after changing `:root` to `[data-md-color-primary=purple]` did the colour update.
I can't tell if this is just outdated documentation or perhaps even unintentional behaviour from the theme, but either way will this require some sort of update here...

**Comment by squidfunk** [comment]
> The issue with that is, that this override does sometimes not work.

Please be more precise as what you mean by "sometimes". If you can provide a case that works and one that doesn't, we can check if the documentation needs to be updated or not, because I may have missed some case, but "sometimes" is too vague.

**Reply by Andre601** [reply]
In my case did I try the above aproach with the `:root` thing while the repository itself uses a theme switcher/selector.

It didn't update the colour itself, but changing `:root` to `[data-md-color-primary=purple]` made it work.

Idk how to better explain it...
